scripts/docs-deploy/snapshot-deploy.ts

```diff
@@ -0,0 +1,72 @@
+import {cloneDocsRepositoryForMajor} from './clone-docs-repo';
+import {performDefaultSnapshotBuild} from '../build-packages-dist';
+import {buildDocsContentPackage} from '../build-docs-content';
+import {installBuiltPackagesInRepo} from './install-built-packages';
+import {DeploymentInfo, deployToSite, isProductionDeployment} from './deploy-to-site';
+import {installDepsAndBuildDocsSite} from './utils';
+import {runMonitoringTests} from './monitoring/index';
```

Comment:
If I had to guess, I would say you're not into alphabetically ordering imports 😛

Reply:
haha, not opposed to it. I usually let VSCode add the imports and we don't enforce a order here. Might be something we can enable in prettier

Reply:
I will let VSCode sort them, but usually this is not worth the time (I consider it still readable 😛), unless we start configuring prettier to do so.